test/fuzz/CMakeLists.txt

```diff
@@ -45,6 +49,36 @@ target_link_libraries(
 
 set(FUZZ_TEST_TARGETS "")
 
+set(DEFAULT_RUNS_NUMBER 10)
```

Comment:
Isn't this amount of runs too small?

Reply:
Yes, this number is tiny by default. This value can be overridden by env variable `RUNS`, see below `-runs= ...`.